### PR TITLE
Adds backend-br/desafios and backend-br/vagas to made-in-brazil collection.

### PR DESCRIPTION
**Pull request by gustavofreze**
### Please confirm this pull request meets the following requirements:

- [X] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [X] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [X] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [X] Content (and my changes are in `index.md`)

>The [backend-br/desafios](https://github.com/backend-br/desafios) repository helps many people who are looking to improve technically, and it does so through technical challenges and discussions about their implementations.
>The [backend-br/vagas](https://github.com/backend-br/vagas) repository helps many people in Brazil to find jobs, it is updated daily. 
>So I thought it would be helpful to list them here.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by gustavofreze** [comment]
@smashwilson

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ _site
 .jekyll-metadata
 vendor
 .bundle
+.idea
```
Comment (octokatherine) [inline .gitignore:9]: why is this change added?

Comment (gustavofreze) [reply .gitignore:9]: to not add cache from Jetbrains IDEs.

**Review by octokatherine — APPROVED** [review]
